contracts/NFTOpt.sol

```diff
@@ -38,26 +42,33 @@ contract NFTOpt {
     /// @notice Address is not owner of the NFT
     error NFT_NOT_OWNER(address presumableOwnerAddress);
 
-    /// @notice Current option state is not allowed for this transaction
-    error INVALID_OPTION_STATE(OptionState currentState, OptionState neededState);
+
+    /// Option event/actions Errors
+    /// @notice Buyer always != seller
+    error CANNOT_HAVE_BUYER_SAME_AS_SELLER();
 
     /// @notice Current timestamp does not allow for option exercise
     error EXERCISE_WINDOW_IS_CLOSED(uint256 expirationTimestamp);
 
     /// @notice Current timestamp does not allow for option exercise
     error OPTION_REQUEST_ALREADY_FULFILLED(address fulfillerAddress);
 
+    /// @notice Current option state is not allowed for this transaction
+    error INVALID_OPTION_STATE(OptionState currentState, OptionState neededState);
+
+
+    /// Funds Transfer Errors
     /// @notice Insufficient funds in escrow to withdrawal
     error INSUFFICIENT_FUNDS();
 
     /// @notice Failed to transfer funds from escrow to msg.sender
     error FUNDS_TRANSFER_FAILED();
 
-    /// @notice Buyer always != seller
-    error CANNOT_HAVE_BUYER_SAME_AS_SELLER();
 
+    // Arithmetic Error

```

Comment:
```suggestion
    /// @notice Arithmetic Error

```

Reply:
I see the other comments as redundant, but still, we should keep it consistent and comments should have @notice before the description

Reply:
sounds good, will add.
curious as to how those macros render, could we get a quick review in standup tomorrow?

Reply:
> sounds good, will add. curious as to how those macros render, could we get a quick review in standup tomorrow?

have a read here: https://blog.soliditylang.org/2021/04/21/custom-errors/

Reply:
*^ thank you!

Reply:
*^ thank you!